tools/theme/components/badge.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentSingleStyleConfig } from '@chakra-ui/react'; // Single component style config
+
+// Badge style config
+const Badge: ComponentSingleStyleConfig = {
+  // Base style
+  baseStyle: {
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: '23px',
+    textTransform: 'inherit',
+    fontWeight: 'normal',
+    fontSize: '16px',
+    height: '32px',
+    paddingX: '1rem',
```

Comment:
I think that we can adjust the left padding a bit here to address the issue of the left part of the badge being too wide.

Reply:
Adjusting it here messed up the positioning for the badges without an icon so I ended up adjusting it in `components/internal/ApplicantStatusBadge.tsx` by changing the left margin value here
```tsx
return (
    <Badge variant={variant}>
      <Box ml="-1" mr="2">
        {_renderBadgeContent(variant)}
      </Box>
    </Badge>
  );
  ```

Reply:
Got it